### PR TITLE
feat(tap): set position to tapped location

### PR DESCRIPTION
**Pull request by conkur**
Now the slider will move to the tapped location.
This adds another way to adjust the slider other than dragging the thumb circle.

Also adds .idea to .gitignore so the whole folder is ignored

### CODE REVIEW

**Comment by conkur** [comment]
Codeowners: I wanted to get your thoughts on this. For my own project, I wanted to add tap capabilities but I'm not sure if it is desired by you. Feel free to close this if this behavior is not desired. Let me know if you'd like a video demonstrating the change.

File: circularSlider/src/main/java/com/github/Mindinventory/circularslider/CircularProgress.kt [hunk 1]
```diff
@@ -53,7 +54,6 @@ import kotlin.math.*
  * @param isDisabled Flag to set enabled/disabled circular slider
  * @param staticProgress Static progress in case if isDisabled is true
  */
-
```
Comment (conkur) [inline circularSlider/src/main/java/com/github/Mindinventory/circularslider/CircularProgress.kt:56]: TODO: Revert this if we want to merge this

**Comment by akashmi** [comment]
@conkur Thanks for using this lib and creating PR with enhancement. will definitely look into this.

**Comment by conkur** [comment]
@akashmi Thanks! Great library, very clean. If I end up finishing my project which is currently using this library I will share my work

**Comment by akashmi** [comment]
@conkur we have included your changes along with a few more enhancements in this PR https://github.com/Mindinventory/AndroidCircularSlider/pull/5

and released the new version 0.0.2. thanks for your contribution.

**Pull request closed without merging**